gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=io.ballerina.stdlib
-version=1.17.1-SNAPSHOT
+version=1.18.0-SNAPSHOT
 ballerinaLangVersion=2201.13.0-20250428-155600-34c3f5da
```

Comment:
Why are we releasing sql with update 13. Do we have any new features or breaking change with update 13?

Reply:
There is one fix https://github.com/ballerina-platform/module-ballerina-sql/pull/777. We don't need to release it. It change only testutils

Reply:
Ack